src/main/java/org/apache/mesos/hdfs/executor/AbstractNodeExecutor.java

```diff
@@ -226,7 +224,7 @@ protected void runCommand(ExecutorDriver driver, Task task, String command) {
     reloadConfig();
     try {
       log.info(String.format("About to run command: %s", command));
-      Process init = Runtime.getRuntime().exec(new String[]{"sh", "-c", command});
+      Process init = new ProcessBuilder("sh", "-c", command).start();
```

Comment:
Instead of method chaining, it is better java style to create a separate process builder object then call start on a separate line.